.env.example

```diff
@@ -56,6 +56,14 @@ CRAFT_DB_PORT=""
 
 
 
+# 
+# IDs/Keys used by third party integrations
+# for production environment
```

Comment:
Why only “for production environment”? We can add a value on any environment, right?

Reply:
Updated the comment.